js/scripts.js

```diff
@@ -1,17 +1,12 @@
-function changeColor(){
-    var hex_numbers = ["0", "1", "2", "3", "4", "5", "6", "7", "8", "9",
-    "A", "B", "C", "D", "E", "F"];
+function changeColor() {
+    var hex_numbers = ["0", "1", "2", "3", "4", "5", "6", "7", "8", "9","A", "B", "C", "D", "E", "F"];
 
     var hexcode = '';
 
-    for(var i=-0;i < 6;i++){
-        var random_index = Math.floor(Math.random() 
-        + hex_numbers.length);
-
-        hexcode += hex_numbers[random_index]
+    for(var i = 0;i <= 5; i++) {
+        var random_index = Math.floor(Math.random() * ((hex_numbers.length -1) - 0 + 1)) + 0;
+        hexcode += hex_numbers[random_index];
      }
- document.getElementById("hex-code").innerHTML
- = hexcode;
- document.getElementsByTagName("body")[0].style.
- background = "#" + hexcode; 
```

Comment:
Just some stylistic changes I made here, but for context: I never separate a declaration & definition (such as you are doing here on lines 13 & 14) and it looks a little cleaner to get the background color assignment the way I did it.